user.proto

```diff
@@ -40,25 +40,25 @@ service Users {
 
 message NewUser {
   string username = 1;
-  string uuid = 2;
-  google.protobuf.Struct attributes = 5;
-  google.protobuf.Struct traits = 6;
+  string external_id = 2;
```

Comment:
Do we want to support a uuid and an external ID? I added the UUID in the scenario where a User ID would be exposed and you don't want to leak the auto-incremented ID. Are you thinking the external ID effectively takes the place of that?

Reply:
On second thought, if someone wants a different ID, we can direct them to use the external IDs and they can generate a UUID or some other ID. We can also add some helpers in our APIs to generate different IDs if that is helpful.